framework/src/modules/interoperability/constants.ts

```diff
@@ -27,7 +27,7 @@ export const REGISTRATION_FEE = 1000000000;
 export const MAX_NUM_VALIDATORS = 199;
 export const MAX_LENGTH_NAME = 40;
 export const MAX_UINT32 = 4294967295;
-export const MAX_UINT64 = 18446744073709551615;
+export const MAX_UINT64 = BigInt('18446744073709551615'); // BigInt((2 ** 64) - 1) - 1
```

Comment:
Do you know where do we use this constant? Because `MAX_UINT32` is number and `MAX_UINT64` is bigint, they are not convertible to each other and this may be a problem for some interfaces depending on the usage.

Reply:
The constant will likely be used for validation of `bigint` in the code.  Of course the usage of `bigint` will be taken into account when it is used.